src/mock_server/bare_server.rs

```diff
@@ -143,7 +143,7 @@ impl BareMockServer {
         &self.server_address
     }
 
-    /// Return a vector with all the requests received by the `BareMockServer` since it started.  
+    /// Return a vector with all the requests received by the `BareMockServer` since it started.
```

Comment:
This whitespace is significant (new line), so let's keep it.

Reply:
Ahh, [EditorConfig](https://editorconfig.org/) is removing trailing whitespace. I fixed by adding a newline.